### PR TITLE
Proj2_GIL

### PR DESCRIPTION
**Pull request by irrit-im**

### CODE REVIEW

File: proj2_GIL/no_impact.py
Comment (simplyCyclops) [inline proj2_GIL/no_impact.py:1]: A. This program doesn't prove that it is not affected by the gil. I need to see a time comparison
B. Challenge: make a multithreaded program that isn't affected by the gil. Prove that it is not affected by running it once with one thead then once multithreaded and show that the times are roughly the same

File: proj2_GIL/bad_impact.py
Comment (simplyCyclops) [inline proj2_GIL/bad_impact.py:1]: In order to prove that the program is slower when multithreaded, measure its duration when run with a signle thread, then compare it to two threads

File: proj2_GIL/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+## Description
+This project contains two programs:
+
+1. `no_impact` is a single-threaded program, and therefore is not affected by the GIL.
+
+3. `bad_impact` is a multi-threaded, CPU-bound program, and therefore is affected by the GIL. This can be proved by comparing its run time to that of a single (identical) thread: the results show that two identical CPU-bound threads take about double the time to run (meaning the GIL does not allow them to run simultaneosly).
```
Comment (simplyCyclops) [inline proj2_GIL/README.md]: 1 then 3 חחחח

File: proj2_GIL/README.md
Comment (simplyCyclops) [inline proj2_GIL/README.md:1]: good readme

File: proj2_GIL/README.md (hunk 1 above)
Comment (irrit-im) [reply proj2_GIL/README.md]: oops
(i was totaly testing you)

File: proj2_GIL/no_impact.py
Comment (simplyCyclops) [inline proj2_GIL/no_impact.py:1]: good, except for the fact that you dont print the multithreaded run time

**Pull request merged**